src/krux/krux_settings.py

```diff
@@ -215,6 +221,8 @@ def label(self, attr):
             "depth_per_pass": t("Depth Per Pass"),
             "part_size": t("Part Size"),
             "border_padding": t("Border Padding"),
+            "head_type": t("Tool head type"),
```

Comment:
Seeing other terms used, I would simplify this one to "Head Type" or "Tool Type" what do you think @odudex and @qlrd ?